rotkehlchen/chain/evm/decoding/decoder.py

```diff
@@ -366,7 +366,6 @@ def decode_by_address_rules(self, context: DecoderContext) -> DecodingOutput:
                 f'Decoding tx log with index {context.tx_log.log_index} of transaction '
                 f'{context.transaction.tx_hash.hex()} through {method.__name__} failed due to {e!s}')  # noqa: E501
             return DEFAULT_DECODING_OUTPUT
-
```

Comment:
don't remove this newline. Separating different returns and indentation with newline for readability is good. Also why touch this file? I guess probably some remnant from changes you did and undid?

Reply:
>I guess probably some remnant from changes you did and undid?

yes